app/src/main/java/com/github/se/project/model/lesson/Lesson.kt

```diff
@@ -1,17 +1,19 @@
 package com.github.se.project.model.lesson
 
+import com.github.se.project.model.profile.Language
 import com.github.se.project.model.profile.Subject
 
 data class Lesson(
     val id: String = "", // Unique identifier for the lesson
     val title: String = "", // Title of the lesson
     val description: String = "", // Description of the lesson
-    val subject: Subject = Subject.PHYSICS,
+    val subject: Subject = Subject.NONE,
+    val languages: List<Language>,
     val tutorUid: String = "", // User ID of the tutor
     val studentUid: String = "", // User ID of the student (if booked)
     val minPrice: Double = 0.0, // Price for the lesson
     val maxPrice: Double = 0.0, // Price for the lesson
+    val price: Double = 0.0, // Price for the lesson
```

Comment:
I know it is not really related to this PR, but I think that we might uses aslo Double for price in the Profile dataclass (for now it is an integer but indeed double seems more appropriates).